import "common.proto";
import "item.proto";

// 请求全部道具 command：301
message GetItemsResponse{
	repeated ItemPB items = 1;
}

message ItemUseResponse
{
    required CommonResponse res = 1;
    optional GameResourcesResponse gain = 2;
}